frontend/src/scenes/settings/environment/CRMUsageMetricsConfig.tsx

```diff
@@ -6,23 +6,45 @@ import {
     LemonButton,
     LemonDialog,
     LemonInput,
+    LemonLabel,
     LemonMenu,
     LemonSelect,
     LemonTable,
     LemonTableColumns,
 } from '@posthog/lemon-ui'
 
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
 import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { TestAccountFilterSwitch } from 'lib/components/TestAccountFiltersSwitch'
 import { LemonField } from 'lib/lemon-ui/LemonField'
 import { LemonTabs } from 'lib/lemon-ui/LemonTabs'
 import { ActionFilter } from 'scenes/insights/filters/ActionFilter/ActionFilter'
 import { MathAvailability } from 'scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow'
 
 import { groupsModel } from '~/models/groupsModel'
-import { EntityTypes } from '~/types'
+import { AnyPropertyFilter, CyclotronJobFiltersType, EntityTypes, FilterType } from '~/types'
 
 import { UsageMetric, crmUsageMetricsConfigLogic } from './crmUsageMetricsConfigLogic'
 
+function sanitizeFilters(filters?: FilterType): Partial<CyclotronJobFiltersType> {
```

Comment:
One final comment here - converting to the CyclotronJobFiltersType is a little weird as again this is super specific to our domain.

From what I can see all your stuff here will just be query based so you can support the normal filtetype instead